functions-methods.js

```diff
@@ -33,4 +54,17 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(emailadress) {
+    if (emailadress.includes('@') && !(emailadress.includes(',')) && !(emailadress.lastIndexOf('.') === emailadress.length - 1)) {
+        return true;
+    }
+    return false;
```

Comment:
Je kunt de uitkomst van de expressie meteen teruggeven:
```suggestion
    return emailadress.includes('@') && !emailadress.includes(',') && emailadress.lastIndexOf('.') !== emailadress.length - 1;
```